src/chage.c

```diff
@@ -66,7 +66,7 @@ static long inactdays;
 static long expdate;
 
 /* local function prototypes */
-static /*@noreturn@*/void usage (int status);
+static /*@noreturn@*/void usage (int status) noreturn_attr;
```

Comment:
C23 specifies that function attributes should go at the beginning of a function.  For future-compatibility with C23's `[[noreturn]]`, I recommend placing this before `static`.

Reply:
Also, you could now remove that comment, I guess.